frontend/src/services/webSocketService.js

```diff
@@ -18,6 +18,10 @@ export default class WebSocketService {
         this.initWebSocket(this.webSocket, lobbyId);
     }
 
+    disconnect() {
+        this.webSocket.close()
```

Comment:
I think you should check if this websocket is open first, so use the `isConnected()` function to check

Reply:
yes sir

Reply:
Do you think I should add an else to do this?
`console.error("NOT CONNECTED");`